devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

```diff
@@ -157,13 +159,12 @@ def configure() {
 	log.debug "configure()..."
 	if (isAeotecHomeEnergyMeter())
 		delayBetween([
-				encap(zwave.configurationV1.configurationSet(parameterNumber: 255, size: 4, scaledConfigurationValue: 1)), // Reset the device to the default settings
-				encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 1)), // report power in Watts...
+				encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 3)), // report total power in Watts and total energy in kWh...
```

Comment:
so this reports power and energy on one association group instead of two, right?

Reply:
That is correct, but this isn't really based on Association Groups, both Parameter 101, 102, and 103 all send to Group Association Group #1 specifically.

Each group just allows you to send various sensors of the HEM Gen5 at different time intervals. (as Group1 interval, Group2 Interval, Group3 Interval)_